docs/source/_templates/layout.html

```diff
@@ -483,5 +483,21 @@ <h2>Resources</h2>
       });
     })
   </script>
+  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+  <script type="text/javascript">
+  let VERSION
+  if ('{{ version }}'.includes('v')) {
+    VERSION = '{{ version }}'
+  } else {
+    VERSION = '{{ release }}'
```

Comment:
`{{ version }}` is from `conf.py`. If a version include `v`, then it's a tag else we will use `release` which is `master` as algolia stated 
```
So as of today you have: master, v0.1.0, v0.1.1, v0.1.2, v0.2.0, v0.2.1, v0.3.0, v0.4.0.post1, v0.4.1, v0.4.2, v0.4.3, v0.4rc.0.post1
```